### PR TITLE
Lower the priority of the initial installation repository (bsc#1071742)

### PR DESCRIPTION
**Pull request by lslezak**
- To prefer the packages from the other media.
- See https://bugzilla.suse.com/show_bug.cgi?id=1071742
- 4.0.27

### CODE REVIEW

File: src/modules/Packages.rb [hunk 1]
```diff
@@ -1741,6 +1741,10 @@ def Initialize_StageInitial(show_popup, base_url, log_url)
         end
       end
 
+      # Lower the priority of the initial installation repository to prefer
+      # the packages from the other media (bsc#1071742)
```
Comment (jreidinger) [inline src/modules/Packages.rb]: Does this cause in opensuse installation to prefer network over DVD when there are identical packages?

Comment (lslezak) [reply src/modules/Packages.rb]: Hmm... good point.

Then the DVD preference would not work correctly... :worried: 

We should do it only when *all* repos are DVD...

Comment (lslezak) [reply src/modules/Packages.rb]: Which means we cannot do it at this place as the addons are added later and we do not know in advance if all will be DVDs...

**Comment by lslezak** [comment]
Closing, we need a different solution for this...

**Pull request closed without merging**